tox.ini

```diff
@@ -9,12 +9,14 @@ sitepackages = true
 deps =
     ansible2.9: ansible~=2.9.0
     ansible2.10: ansible~=2.10.0
-    ansible-lint
+    ansible3: ansible~=3.0.0
```

Comment:
Update playbooks/example-{use,source}/playbook.yml also with the version check for 2.9